easycv/datasets/classification/pipelines/auto_augment.py

```diff
@@ -1043,3 +1087,86 @@ def __repr__(self):
         repr_str += f'pad_val={self.pad_val}, '
         repr_str += f'prob={self.prob})'
         return repr_str
+
+
+@PIPELINES.register_module()
+class gaussianblur(object):
```

Comment:
we have already register `GaussianBlur` from torchvision, please refer to: `easycv/datasets/shared/pipelines/third_transforms_wrapper.py`

Reply:
done